src/patternfly/components/FormControl/form-control.scss

```diff
@@ -95,6 +115,32 @@ $pf-c-form-control--m-search--Coordinates: "M505 442.7L405.3 343c-4.5-4.5-10.6-7
   --pf-c-form-control--m-search--BackgroundSize: var(--pf-c-form-control--FontSize) var(--pf-c-form-control--FontSize);
   --pf-c-form-control--m-search--BackgroundUrl: #{pf-bg-svg($pf-c-form-control--m-search--Coordinates, $svg-color: $pf-c-form-control--m-search--Color)};
 
+  // Input m-icon
+  --pf-c-form-control--m-icon--PaddingRight: calc(var(--pf-c-form-control--inset--base) + var(--pf-c-form-control--m-icon--BackgroundSizeX) + var(--pf-c-form-control--m-icon--icon--spacer));
+  --pf-c-form-control--m-icon--BackgroundUrl: none;
+  --pf-c-form-control--m-icon--BackgroundPositionX: calc(100% - var(--pf-c-form-control--inset--base));
+  --pf-c-form-control--m-icon--BackgroundPositionY: center;
```

Comment:
Looks like, where `--BackgroundPositionY: center` affects the `textarea` s by aligning to vertical center: 

![Screen Shot 2020-09-08 at 9 38 45 AM](https://user-images.githubusercontent.com/5385435/92484654-f631f600-f1b7-11ea-913f-d5ddbdf447ae.png)

Looks like reusing 
```
--pf-c-form-control--m-icon--BackgroundPositionY: var(--pf-c-form-control--PaddingTop);
```
or using a `--base` value retains vertical positioning.

Reply:
doh! Good catch @mattnolting changed it back to what it was before with a note to use the `--pf-c-form-control--inset--base` var in a breaking change release.